src/main/java/org/sagebionetworks/bridge/AuthUtils.java

```diff
@@ -21,20 +21,18 @@
  * All methods throw UnauthorizedException if they fail.
  */
 public class AuthUtils {
+    
+    public static final AuthEvaluator CAN_READ_ORGANIZATIONS = new AuthEvaluator()
+            .hasAnyRole(ADMIN);
+
     /**
      * Is this scoped to specific studies? It should have one of the study-scoped
      * roles, and no roles that are app scoped that we would allow wider latitude
      * when using the APIs.
      */
     public static final AuthEvaluator CAN_READ_ORG_SPONSORED_STUDIES = new AuthEvaluator()
-            .hasAnyRole(ORG_ADMIN, STUDY_COORDINATOR).hasNoRole(DEVELOPER, RESEARCHER, ADMIN, WORKER);
+            .hasAnyRole(ORG_ADMIN, STUDY_DESIGNER, STUDY_COORDINATOR).hasNoRole(DEVELOPER, RESEARCHER, ADMIN, WORKER);
 
-    /**
-     * Can the caller delete an organization?
-     */
-    public static final AuthEvaluator CAN_DELETE_ORG = new AuthEvaluator()
-            .hasAnyRole(ADMIN);
```

Comment:
Why add CAN_READ_ORGANIZATION and then remove this?

Reply:
It was removed because its one condition is easily verified by getAuthenticatedSession(ADMIN) in the controller. CAN_READ_ORGANIZATIONS is just an admin check so that's what I'm going to use there, too.